nh_eobs_mental_health/models/nh_clinical_pme_obs_stop.py

```diff
@@ -149,12 +149,17 @@ def create_new_ews(self):
 
         self.force_v7_api(activity_model)
```

Comment:
Now we've broken the `nh_activity.data_model_event()` into v7 & v8 is it possible to remove the reliance on converting to v7?